src/components/entry/Entry.jsx

```diff
@@ -11,6 +19,36 @@ export default function Entry({ entry }) {
 				lng: {entry.location[0]} <br />
 				lat: {entry.location[1]}
 			</Typography>
+			<RenderIf isTrue={!isDeleteMode}>
+				<Button
+					type="button"
+					variant="smallDo"
+					onClick={() => {
+						toggleMode(isDeleteMode, setIsDeleteMode);
```

Comment:
this toggleMode function does not help you at all:

```suggestion
						setIsDeleteMode(!isDeleteMode);
```